lte/gateway/c/core/oai/tasks/nas/esm/DefaultEpsBearerContextActivation.c

```diff
@@ -357,8 +358,8 @@ status_code_e esm_proc_default_eps_bearer_context_reject(
     ue_mm_context_t* ue_context_p =
         PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
     if (!ue_context_p) {
-      OAILOG_ERROR(
-          LOG_NAS_ESM,
+      OAILOG_ERROR_UE(
```

Comment:
this will also complain since there is no `ue_context_p` retrieved, let's not change this

Reply:
Resolved

Reply:
Resolved